src/Resolvers/ContainerResolver.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OWC\Zaaksysteem\Resolvers;
+
+use DI\Container;
+use OWC\Zaaksysteem\Contracts\Client;
+use OWC\Zaaksysteem\Foundation\Plugin;
+
+class ContainerResolver
```

Comment:
Is het een `Container` of `Client` resolver?

Reply:
Wel een container, het ophalen van het RSIN nummer is nu zo gedaan:
`ContainerResolver::make()->get('rsin')`